Assets/Tests/InputSystem/Plugins/WebGLTests.cs

```diff
@@ -40,11 +40,24 @@ public void Devices_SupportsWebGLStandardGamepads()
         });
         InputSystem.Update();
 
-        Assert.That(gamepad.leftStick.ReadUnprocessedValue(), Is.EqualTo(new Vector2(0.345f, -0.456f))); // Y inverted on WebGL.
-        Assert.That(gamepad.rightStick.ReadUnprocessedValue(), Is.EqualTo(new Vector2(0.567f, -0.678f))); // Y inverted on WebGL.
         Assert.That(gamepad.leftTrigger.ReadUnprocessedValue(), Is.EqualTo(0.123).Within(0.0001));
         Assert.That(gamepad.rightTrigger.ReadUnprocessedValue(), Is.EqualTo(0.234).Within(0.0001));
 
+        AssertStickValues(gamepad.leftStick, new Vector2(0.345f, -0.456f), -0.456f, 0, 0, 0.345f);
+        AssertStickValues(gamepad.rightStick, new Vector2(0.567f, -0.678f), -0.678f, 0, 0, 0.567f);
+
```

Comment:
Minor: Any reason for these double line breaks?

Reply:
It's what the formatter decided it wanted. Who am I to argue with the formatter :)